File: Perl/scripts/Problem006.pl
sub squareSum {
    my $n = @_[0];    
    ( ( $n*($n+1)/2 )**2 )
}

sub sumSquare {
    my $n = @_[0];
    ( $n * ($n+1) * (2*$n + 1) / 6 )
}

$start = time();
$ans = squareSum(100) - sumSquare(100);
$exc_time = time() - $start;
print "\nAnswer: $ans\n";
print "Time Taken: $exc_time seconds\n\n";

File: Python/scripts/Problem006.py
from time import time

start = time()
sqSum = lambda n: int(n * (n+1) * (2*n + 1) / 6)
sumSq = lambda n: int(n * (n+1) / 2) ** 2

ans = sumSq(100) - sqSum(100)

if __name__ == '__main__':
    print(f"\nAnswer: { ans }")
    print(f"Time Taken: { time() - start }\n")
